sci-misc/boinc/files/boinc.init

```diff
@@ -23,30 +23,45 @@ create_work_directory() {
 		chown "${USER}:${GROUP}" "${RUNTIMEDIR}"
 	fi
 
-	if [[ ! -e "${RUNTIMEDIR}"/ca-bundle.crt ]]; then
+	if [ ! -e "${RUNTIMEDIR}"/ca-bundle.crt ]; then
 		ln -s /etc/ssl/certs/ca-certificates.crt "${RUNTIMEDIR}"/ca-bundle.crt
 	fi
 
 	return 0
 }
 
 
+fix_lib_symlinks() {
+	local src="$1"
+	local tgt="$2"
+
+	if test -n "${tgt}"; then
+		# Remove a broken symlink
+		if test -h "${tgt}" \
+			-a "${src}" != "$(readlink "${tgt}")" ; then
```

Comment:
Strictly speaking, `readlink` is not POSIX. And before you ask — no, there's no POSIX-compliant way to read symlinks from shell.